cobbler/tftpgen.py

```diff
@@ -26,7 +26,7 @@
 import os.path
 import re
 import socket
-import string
+#import string
```

Comment:
Travis reports the following: `cobbler/tftpgen.py:29:1: E265 block comment should start with '# '`. I would say let us remove it if it is not needed.

Reply:
Ok, got it.